readthedocs/api/base.py

```diff
@@ -140,7 +140,7 @@ class Meta(object):
     #     return bundle
 
     def get_object_list(self, request):
-        self._meta.queryset = Version.objects.api(user=request.user, only_active=False)
+        self._meta.queryset = Version.objects.api(user=request.user)
```

Comment:
Why did you remove the only_active here? Seems like we want to include all objects, not just active.